scripts/generate/public_source_gen.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+# Copyright (c) F5, Inc.
+# This source code is licensed under the Apache License, Version 2.0 license found in the
+# LICENSE file in the root directory of this source tree.
+
+# shellcheck shell=bash
+# Generate support file from a public git repo
+# Only call it through //go:generate comments in project root to update .gen.go files
+
+set -e
+
+branch="master"
+url=""
+output_path=""
+sub_path=""
+config_path=""
+
+help() {
+    cat << EOF
+Generate support file from a public repo(currently only git).
+Only call me through //go:generate comments in project root to update .gen.go files.
```

Comment:
Nit: I don't think the help text needs to say this. `//go:generate` is just a good way to do it as the `analyzer.go` file is basically documenting a dependency on those files.

Reply:
Since we have `go run ./cmd/generate "${genArgs[@]}"` in shell script it can only run at project root. Do we want to put the path of ` ./cmd/generate` as an  param? It's more flexible but a new required para for the script

Reply:
No, I don't think we need that.

Reply:
Didn't change: retain the comments